examples/test_snet_sdk_agent.metta

```diff
@@ -0,0 +1,14 @@
+!(import! &self motto)
+!(bind! &agent1
+    (
+     snet-sdk-agent naint paraphrase-generation
+          ;set method name and parameters for it
+         (py-dict ((method paraphrase) (sentence (query))))
+         (py-dict ((free_call_auth_token_bin
+          "35c9a13282fb034f8caafae26910d0e9fe6e8e237df73fe9d999df8386df339408cd75c93d759f2ad5848d31005cd48e9b4257908f5cbce05b2de6aa8f929d941b")
```

Comment:
It would be better to avoid putting a free call token here (it will expire after all), but leave a comment that it should be provided via environment variables.

Reply:
free_call_auth_token_bin is an optional argument should I remove it from snet-sdk-agent ?

Reply:
It shouldn't be removed, but stay optional in sense that if someone wants to pass it as a parameter to `snet-sdk-agent`, it should work. Does it work like this right now?

Reply:
> Does it work like this right now?
yes.